examples/c/wind_turbine_nmpc.c

```diff
@@ -667,8 +760,14 @@ int main()
 		}
 		else if (plan->sim_solver_plan[i].sim_solver == NEW_LIFTED_IRK)
 		{
-			sim_opts->ns = 4;
-			sim_opts->num_steps = 1;
+			sim_opts->ns = 5;
+			sim_opts->num_steps = 4;
+		}
+		else if (plan->sim_solver_plan[i].sim_solver == GNSF)
+		{
+			sim_opts->ns = gnsf_num_stages;
+			sim_opts->num_steps = gnsf_num_steps;
+			sim_opts->jac_reuse = true;
```

Comment:
cool, did you already implemented that? ;)

Reply:
jacobian reuse?
yes, i worked on this yesterday, see this commit [jac_reuse](0adb13badf9b5f55dce2ef3c948b396bce743469)